inception-project/inception
# Refactoring/1033 refactor prediction task.java

**What's in the PR**
* Refactoring task - PredictionTask.java

**How to test manually**
* Start inception
* Login
* Import Project
* Go to annotation screen
* screen should load the annotations and active learning tab should work
